<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QMainWindow" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#game{border-image: url(:/pig/pigfarm.jpg);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblNowZhuchang">
            <property name="styleSheet">
             <string notr="true">QLabel
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:30px
}</string>
            </property>
            <property name="text">
             <string>现有猪场数量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNowMoney">
            <property name="styleSheet">
             <string notr="true">QLabel
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:30px
}</string>
            </property>
            <property name="text">
             <string>现有资金：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btnShop">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
            </property>
            <property name="text">
             <string>商店</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExit">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
            </property>
            <property name="text">
             <string>退出登陆</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lblNowCount">
            <property name="styleSheet">
             <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:30px
}</string>
            </property>
            <property name="text">
             <string>现有小猪数量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNowFood">
            <property name="styleSheet">
             <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:30px
}</string>
            </property>
            <property name="text">
             <string>现有食物：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblMan">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btnZhuchang1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>猪场01</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnZhuchang2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>猪场02</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnZhuchang3">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>猪场03</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnZhuchang4">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>猪场04</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnZhuchang5">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>猪场05</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnZhuchang6">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>猪场06</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblTime">
      <property name="maximumSize">
       <size>
        <width>55555</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color:red;
font: 11pt &quot;方正舒体&quot;;
font-size:40px
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
